controllers/argocd/dex.go

```diff
@@ -160,10 +159,32 @@ func (r *ReconcileArgoCD) getOpenShiftDexConfig(cr *argoproj.ArgoCD) (string, er
 	dex := make(map[string]interface{})
 	dex["connectors"] = connectors
 
+	if err := updateDexConfig(cr, dex); err != nil {
```

Comment:
Couldn't hurt to add a quick comment here describing what `updateDexConfig` is doing (or update the function name to better indicate its purpose, if that makes sense)

Reply:
updated the name .